docs/api/api.yml

```diff
@@ -0,0 +1,48 @@
+openapi: "3.0.3"
+
+info:
+  title: "JongReki API"
+  version: "1.0.0"
+
+servers:
+  - url: "http://localhost:5555"
+    description: "development server"
+paths:
+  "/friends":
+    get:
+      summary: "Get all friends"
+      description: "Get data from the friends table including all status."
+      responses:
+        "200":
+          description: "Get all friends successfully"
+          content:
+            application/json:
+              schema:
```

Comment:
It seems that sometimes only $ref: '#/components/schemas/User' is written. 
How detailed should it be described?

Reply:
Hi @kakimaru 
I've added some use cases of `$ref` so u can refer them.
and I've added following changes
- Added versioning strategy in OpenAPI spec
- Created reusable components for common responses
- Added tags for better organization
- Fixed data types to match our DB schema (user_id is now integer 🔧)

Let's talk about this tomorrow.